ocaml/xapi-cli-server/records.ml

```diff
@@ -1460,6 +1460,17 @@ let pool_record rpc session_id pool =
               ~value:(bool_of_string x)
           )
           ()
+      ; make_field ~name:"telemetry-interval"
+          ~get:(fun () ->
+            (x ()).API.pool_telemetry_interval
+            |> Record_util.telemetry_frequency_to_string
```

Comment:
This is again mixing frequency with intervals - it can be only one. Obviously they are related but the nomenclature is using both without good reason.

Reply:
Use `frequency` now. Thanks.